### PR TITLE
Umbraco Commerce v14 Docs Updates

### PR DESCRIPTION
**Pull request by mattbrailsford**
## Description

Updated docs with changes from v14.

This PR should cover updated screenshots, removal of old API examples, and addition of docs for any new APIs

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Umbraco Commerce v14

## Deadline (if relevant)

ASAP

### CODE REVIEW

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Thanks for the PR, @mattbrailsford 🙌 I will incorporate most of the suggestions related to spelling/typos, styleguide, and so on.

Can you take a look at the rest of the suggestions - shortened couple of sentences to fix the vale error, is WIP Note required, and add Management API in the Summary file?

File: 14/umbraco-commerce/key-concepts/discount-rules-and-rewards.md [hunk 1]
```diff
@@ -156,25 +150,71 @@ result.SubtotalPriceAdjustments.Add(new DiscountAdjustment(ctx.Discount, price))
...
+    alias: "My.PropertyEditorUi.MyDiscountRuleLabel",
+    name: "My Discount Rule Label",
+    element: () => import('./my-discount-rule-label.element.js')
+  };
 
+  export const manifests = [ myDiscountRuleLabelManifest ];
 ```
+
+{% hint style="info" %}
+Our component is registered as a Property Editor UI, but without a defined scheme, it won't display in Umbraco backoffice as a pickable property editor for use on Document Types.
```
Comment (eshanrnh) [inline 14/umbraco-commerce/key-concepts/discount-rules-and-rewards.md:200]:
```suggestion
Without a defined scheme, the Property Editor UI will not display in Umbraco's backoffice as a pickable property editor for use on Document Types.
```

**Comment by mattbrailsford** [comment]
I think that should be everything taken care of, but please let me know if I've missed anything

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @mattbrailsford 🙌 Thanks for accepting the suggestion 💪

**Pull request merged**